src/main/ruby/post-boot/post-boot.rb

```diff
@@ -112,3 +112,7 @@
     end
   end
 end
+
+Truffle::Boot.delay do
+  Truffle::Patching.install_local_patches
+end
```

Comment:
This should be higher, above `# Post-boot patching when using context pre-initialization` as that should be last.
It probably also only makes sense if `home` is set.

Reply:
`Truffle::Patching` will not be defined with `-Xpatching=false`, so it should be `if Truffle::Boot.get_option('patching')`

Reply:
But looks fine here otherwise.